.github/pull_request_template.md

```diff
@@ -1,47 +1,43 @@
 Resolves #ISSUE(s).
-<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address the issue, you can use one of these patterns instead to simply link to the issue and not auto-close it:
-
-Partially addresses:
-- ISSUE(s)
-
-Addresses sub-task (n) in:
-- ISSUE(s)
+<!--- "Resolves #ISSUE" will automatically close #ISSUE when the PR is merged. However, if this PR won't 100% address 
+the issue and you don't want it to auto-close, you can write one of the following instead of "Resolves"
+- Partially addresses
+- Addresses sub-task (n) in
 -->
 
-
-## Pre-merge checklist
-
-<!--- Docs: A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
-
-### PR Overview
-<!--- Short paragraph or short bulleted list describing changes in this PR -->
-
+## Overview
+<!--- A few sentences describing changes / what problem is solved. Should describe the expected result of the changes. 
```

Comment:
## Thoughts about "Additional details"
I didn't add a change, but just wanted to share a thought. The template as it is doesn't create sections for 100% of what a user may want to right. Perhaps that's fine. If a user wants to, they can simply add additional text / sections. For example, I often have an "additional details" section, or a "change details" section where I expand further upon the overview section. Here I might add additional thoughts or link to related issues / PRs.

If we want, we can add a comment to the bottom or a blank "additional details" section, or we can just not worry about it.

Reply:
Good point, I think it's fine if one of us has more to add to just add a section when needed.